src/main/java/pl/psnc/dei/ui/components/imports/CreateImportComponent.java

```diff
@@ -111,12 +111,14 @@ private void refresh() {
 		switchingTables.setWidthFull();
```

Comment:
Line 75 - remove this useless assignment - projectsRepository is already assign in line 69
Lines 90-91 Can be simplified